hashpy/io/obspyIO.py

```diff
@@ -63,21 +63,22 @@ def inputOBSPY(hp, event):
 	if arrv.phase not in 'Pp':
 	    continue
 	
-	if (pick.polarity is 'positive'):
-	    hp.p_pol[k] = 1
-	elif (pick.polarity is 'negative'):
-	    hp.p_pol[k] = -1
-	else:
-	    continue
-	
-	if  (pick.onset is 'impulsive'):
+	if  (pick.onset == 'impulsive'):
```

Comment:
@megies is correct. use `==` to compare strings, never `is`.
`is` is a much stricter and here unwanted comparison.